config/500-controller.yaml

```diff
@@ -78,5 +72,8 @@ spec:
       - name: config-logging
         configMap:
           name: config-logging
+      - name: google-cloud-key
+        secret:
+          secretName: google-cloud-key
```

Comment:
Yes, it will have to be. I need to update the documentation. 
In order for the controller not to crash loop, we can provide a dummy "empty" secret, that the operator will then need to patch.

Reply:
Planning on adding documentation in follow ups if you are OK with it.

Reply:
That is not needed in OSS, documentation should say that the secret should be created first, and then the yamls should be applied.

Reply:
I'm fine with doing docs as a followup, just create an issue for it and resolve this conversation.

Reply:
Done! https://github.com/google/knative-gcp/issues/456
thanks!